src/base/_defaults.scss

```diff
@@ -171,6 +172,25 @@ section {
   display: block;
 }
 
+/**
+ * Shared indentation for typographic blocks
+ */
+
+ul,
+ol,
+blockquote {
+  padding-left: sizes.$type-block-indent;
+}
+
+/**
+ * Adjacent children of blockquotes should inherit parent `margin-top` values
+ * to maintain vertical rhythm.
+ */
```

Comment:
Perhaps, worth a link in the comment block to a reference explaining the lobotomized owl selector?

https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/

Reply:
Good idea. I'm also noticing a slight inaccuracy in my comment that's a holdover from an abandoned technique. Updating...

Reply:
Updated!